testriskscores.js

```diff
@@ -0,0 +1,87 @@
+const { calculateRiskScore } = require("./utils");
```

Comment:
why is the test file along side all the other files, also, could you elaborate why you chose this specific folder structure for the project?

Reply:
Had planned initially to do it but deprioritised it for getting the service up and running.